1-foundations/2-control-flows-and-functions/foundations-s2.ipynb

```diff
@@ -801,6 +824,117 @@
     "For instance, in the first iteration, the day is set to **Mon**. None of the if or elif statements holds True for Monday. So the location is set to whatever is specified in else block. In the next iteration, the day is set to **Tue**. Here the second elif block satisfies the condition, and so location is set by that code block. "
    ]
   },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "f41f93f4",
+   "metadata": {},
+   "source": [
+    "#### `range()` Function"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "d319d66f",
+   "metadata": {},
+   "source": [
+    "Range function gives us a continuous iterable of integers as defined by the start and end values. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 20,
+   "id": "8e22e7f4",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "range(0, 10) \n",
+      "\n",
+      "Output for range(0,10)\n",
+      "0\n",
+      "1\n",
+      "2\n",
+      "3\n",
+      "4\n",
+      "5\n",
+      "6\n",
+      "7\n",
+      "8\n",
+      "9\n",
+      "\n",
+      "Output for range(10)\n",
+      "0\n",
+      "1\n",
+      "2\n",
+      "3\n",
+      "4\n",
+      "5\n",
+      "6\n",
+      "7\n",
+      "8\n",
+      "9\n"
+     ]
+    }
+   ],
+   "source": [
+    "## For instance range(0, 10) produces a sequqnce of integers from 0 to 10. [0 - included, 10 - excluded]\n",
+    "\n",
+    "print(range(0,10), '\\n')\n",
```

Comment:
First time we use `\n` notation. I suggest either explain or do something less elegant like:
`
print("")
print(range(0,10))
`